### PR TITLE
1903 - hzn mms object download does not handle existing file correctly

### PR DESCRIPTION
**Pull request by sirotsinskuy**

### CODE REVIEW

File: cli/sync_service/download.go [hunk 1]
```diff
@@ -55,6 +55,8 @@ func ObjectDownLoad(org string, userPw string, objType string, objId string, fil
 					filePath = filePath + "/"
 				}
 				fileName = fmt.Sprintf("%s%s_%s", filePath, objType, objId)
+			} else {
+				fileName = filePath
```
Comment (dabooz) [inline cli/sync_service/download.go:59]: I think you will need to update the code starting at https://github.com/open-horizon/anax/blob/master/cli/sync_service/download.go#L62 in order to handle the case where the file already exists, including a new message and a flag to overwrite the existing file as per the issue description.

Comment (sirotsinskuy) [reply cli/sync_service/download.go:59]: Sure. Updated PR with adding the new flag.
Previously I was only fixing a bug (that's first part of issue - "At the very least the error msg should say the file already exists"), so after my previous change user would receive that message. I thought that if this issue has an `bug` label we'll need only fix the bug, and if we need to implement a new flat - there will be one more `feature` issue for that.

File: cli/sync_service/download.go [hunk 2]
```diff
@@ -55,12 +55,16 @@ func ObjectDownLoad(org string, userPw string, objType string, objId string, fil
...
 			}
 		}
 	}
 
-	if _, err := os.Stat(fileName); err == nil {
-		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("File %s already exists. Please specify different file path or file name. Or remove file and retry object download", fileName))
+	if !overwrite {
+		if _, err := os.Stat(fileName); err == nil {
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("File %s already exists. Please specify different file path or file name. Or use `overwrite` flag to overwrite existing file.", fileName))
```
Comment (dabooz) [inline cli/sync_service/download.go:66]: small tweak to the message, please change to:
File %s already exists. Please specify a different file path or file name. To overwrite the existing file, use the '--overwrite' flag.

File: cli/sync_service/download.go (hunk 1 above)
Comment (dabooz) [reply cli/sync_service/download.go:59]: I understand. In this project, we don't make a strong distinction between bug and feature, especially when it comes to user facing interfaces like the CLI.

File: cli/sync_service/download.go (hunk 2 above)
Comment (sirotsinskuy) [reply cli/sync_service/download.go]: Sure, message has been updated

**Pull request merged**